misk/src/main/kotlin/misk/web/AdminTabModule.kt

```diff
@@ -15,22 +17,37 @@ import misk.web.proxy.WebProxyEntry
  * Config tab is tightly coupled to the config module. Thus binding should be in ConfigWebModule
  */
 
-class AdminTabModule : KAbstractModule() {
+class AdminTabModule(val environment: Environment) : KAbstractModule() {
   override fun configure() {
-    multibind<WebProxyEntry>().toInstance(
-        WebProxyEntry("/_admin", "http://localhost:3100/"))
-    multibind<WebActionEntry>().toInstance(
-        WebActionEntry<WebProxyAction>("/_admin"))
-
-    multibind<WebProxyEntry>().toInstance(
-        WebProxyEntry("/_tab/dashboard", "http://localhost:3110/"))
-    multibind<WebActionEntry>().toInstance(
-        WebActionEntry<WebProxyAction>("/_tab/dashboard"))
-
-    multibind<WebProxyEntry>().toInstance(
-        WebProxyEntry("/@misk", "http://localhost:9100/"))
-    multibind<WebActionEntry>().toInstance(
-        WebActionEntry<WebProxyAction>("/@misk"))
+
+
+    if (environment == Environment.DEVELOPMENT) {
+      multibind<WebActionEntry>().toInstance(
+          WebActionEntry<WebProxyAction>("/_admin/"))
+      multibind<WebActionEntry>().toInstance(
+          WebActionEntry<WebProxyAction>("/_tab/dashboard/"))
+//    multibind<WebActionEntry>().toInstance(
```

Comment:
nit: we don’t check in commented-out code. Instead, delete it & use git history. Or if it’s coming back soon you can use `if (false)`. The benefit of `if (false)` is that it survives refactoring.